client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -76,35 +76,38 @@ const ProfileInfoEdit = () => {
             </Form.Text>
           </Form.Group>
 
-          <Form.Group controlId="formResearchGroupName">
+          <Form.Group>
             <Form.Label>Research Group Name</Form.Label>
             <Form.Control
+              id="placeholder-text"
```

Comment:
I couldn't figure out how to get the style to show up using classNames, so it might not be an attribute of Form.Control. If anyone knows a better way, please let me know!

Reply:
Interesting... I believe a css class selector should achieve basically the same thing as the `id` attribute - allowing you to target an element, or a set of common elements for styling, so I wonder what makes this special. Do you have any thoughts @pmyip3 @mable-yip ?